src/utils.ts

```diff
@@ -1452,12 +1477,13 @@ export class Mark extends newShape {
   }
 
   public buildPath(): Path2D {
-    const path = this.path;
+    const path = new Path2D();
     const halfSize = this.size / 2;
     path.moveTo(this.center.x - halfSize, this.center.y);
     path.lineTo(this.center.x + halfSize, this.center.y);
     path.moveTo(this.center.x, this.center.y - halfSize);
     path.lineTo(this.center.x, this.center.y + halfSize);
+    this.path = path;
```

Comment:
If we go for this solution, why not doing this : 

```ts
this.path = new Path2D();
const halfSize = this.size / 2;
path.moveTo(this.center.x - halfSize, this.center.y);
path.lineTo(this.center.x + halfSize, this.center.y);
path.moveTo(this.center.x, this.center.y - halfSize);
path.lineTo(this.center.x, this.center.y + halfSize);
```

Reply:
sure

Reply:
Harmonized everything